text/0006-architecture-review-process.md

```diff
@@ -11,27 +11,29 @@ layout: text
 
 ## Introduction
 
-Architectural Reviews within the Firefox organization fall into two categories: "Roadmap" and "Design".
+Architectural Reviews within the Firefox organization fall into two categories: “Roadmap” and “Design”.
 
 Product Reviews are not within the scope of this process. This process is about architectural change, which should ultimately be in support of product features, but will generally have an effect on multiple product features. Architectural change can be growing new capabilities or in reducing large-scale technical debt.
 
-The function of a **Roadmap Review** is to decide if a thing should be done. The goal is to bring together a packet of data to inform a management decision to provide resources to make the thing happen. A Roadmap Review should happen early in the process so that build time isn’t wasted on a "No" decision, but so that enough information is available to management.
+The function of a **Roadmap Review** is to decide if a thing should be done. The goal is to bring together a packet of data to inform a management decision to provide resources to make the thing happen. A Roadmap Review should happen early in the process so that build time isn’t wasted on a “No” decision, but so that enough information is available to management.
 
 A **Design Review** is to decide if the thing should be done in a particular way. The goal is to ensure that the problem space is understood well enough that a realistic bug breakdown can be made, that those working on the problem all agree on the challenge, and that work can be prioritized to hit the hardest parts of the problem first (in order to get early warning if the hard parts prove impossible or to inform schedule projections early).
 
 ## Timetable
 
 For both a Roadmap Review and a Design Review the **timetable** is as follows:
 
-1. The team should set the stage (see "initial meeting" below):
-  * Summarize in a one or two sentences the proposal to be reviewed. This should be agreed between the team asking for review and the chair.
-  * Find someone to **chair** the review:
+1. The team should set the stage:
+  * Summarize in a one or two sentences the proposal to be reviewed. This should be agreed between the team asking for review and the reviewer.
+  * Find a **reviewer**:
     - For a Roadmap Review this will be the least senior manager with the ability to address the problem in its entirety.
     - For a Design Review this will be an experienced engineer in the problem domain outside of the team willing to put time into the problem. As engineers don’t review their own code they shouldn’t review their own designs. The principle is to get the most informed and least biased feedback possible.
+  * Optionally, for larger reviews, find someone to **chair** the review. This enables to reviewer to concentrate on the problem rather than the admin of the review.
```

Comment:
Nit: the phrase "admin of the review" is a bit confusing. If I understand correctly what you're saying, changing it to "administration of the review" would be sufficient.

It might also make sense to clarify that the role of the chair, as distinct from that of the reviewer, is simply to facilitate the process (i.e. schedule the meeting, manage the clock, ensure minutes are taken, etc.).

I also wonder if it's worth elaborating on the difference between a smaller and a larger review. The separation of the roles of review and chair is one difference, but presumably there are others. Ideally, a "small review" is something that can be thrown together quickly and with barely more than a single person "team," a proposal, and a single reviewer.

Reply:
Clarified wording in 36ceb71af34aafdda70b939e1d2c58eca0f79dea.

The point about differences between a smaller and a larger reviews is made about line 64:

> The ideal review process scales well. The same basic system should work for a quick 2 person decision over the best way to design a certain feature, and for a critical organization-wide decision about a path to take. The term “team” is used above although we strongly recommend design reviews for smaller questions. If you feel yourself wondering if some design is best, it should be easy to perform a review.

Other than the separation of chair/reviewer, most of the changes that I'm aware of would be simply scale, but maybe we need more experience to be sure.